backend/file-orchestrator-service/api/src/main/kotlin/Core.kt

```diff
@@ -518,4 +518,6 @@ data class FSFileSupport(
 
     @UCloudApiDoc("Declares support for the streamingSearch endpoint")
     val streamingSearchSupported: Boolean = false,
+
+    val sharesSupported: Boolean = true
```

Comment:
At this point in time, it would be a more safe bet to default to false. We risk accidentally enabling this on Hippo.